File: JPW_Changed/JPW_Backend - Copy/Models/ApplyforJob.cs
using System;
using System.Collections.Generic;

namespace JPW.Models
{
    public partial class ApplyforJob
    {
        public int AppliedJobId { get; set; }
        public int JobSeekerId { get; set; }
        public int JobId { get; set; }
        
        //public byte[]? Resume { get; set; }
       public string Resume {get; set;}

        public virtual JobPosting Job { get; set; } = null!;
        public virtual JobSeeker JobSeeker { get; set; } = null!;
    }
}

File: JPW_Changed/JPW_Frontend - Copy/src/app/applyforjob.ts
// export class Applyforjob {
//     appliedJobId!:number;
//     jobSeekerId!: number;
//     jobId!: number;
//     resume!: File
// }


export class Applyforjob {
    appliedJobId?: number;
    jobSeekerId?: number;
    jobId?: number;
    resume?: File | null; // Allow null for initial form state
    isApplied: boolean | undefined;
  }
  